__tests__/index.test.ts

```diff
@@ -896,13 +908,15 @@ describe('Auth0', () => {
     });
 
     describe('when there is a valid query string in the url', () => {
-      const localSetup = async () => {
+      const localSetup = async (
+        clientOptions: Partial<Auth0ClientOptions> = {}
```

Comment:
Let's make this optional rather than providing an object by default.

```suggestion
        clientOptions?: Partial<Auth0ClientOptions>
```